azure-pipelines/artifacts/_stage_all.ps1

```diff
@@ -7,14 +7,7 @@ param (
 )
 
 $RepoRoot = [System.IO.Path]::GetFullPath((Join-Path $PSScriptRoot (Join-Path .. ..)))
```

Comment:
Resolve-Path "$PSScriptRoot/../.." this should be an alternative to the code above.